src/main/java/com/iridium/iridiumskyblock/listeners/BlockPlaceListener.java

```diff
@@ -33,13 +33,24 @@ public void onBlockPlaceEvent(BlockPlaceEvent event) {
             }
             return;
         }
-
         XMaterial material = XMaterial.matchXMaterial(event.getBlock().getType());
         if (!IridiumSkyblock.getInstance().getIslandManager().getIslandPermission(island.get(), user, PermissionType.BLOCK_PLACE)) {
             event.setCancelled(true);
-            player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().cannotPlaceBlocks.replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)));
+            player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().cannotPlaceBlocks
```

Comment:
I dont like this formatting

Reply:
oof

Reply:
others is fine?

Reply:
I prefer things to be on one line unless there are alot of .replace's